lib/dor/text_extraction/abbyy/ticket.rb

```diff
@@ -16,10 +16,14 @@ def write_xml
           File.write(file_path, xml)
         end
 
+        def file_path
```

Comment:
the file path for the ticket seemed reasonable to make public, and simplified the test a bit